frontend/src/components/HospitalInventoryEdit.tsx

```diff
@@ -0,0 +1,194 @@
+              value={hospitalAddress[0] ? hospitalAddress[0].zip : ''}
+              onChange={handleAddressChange}
+            />
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} xs={8} md={6}>
+            <Form.Control as="select" 
+              name="country"
+              value={hospitalAddress[0] ? hospitalAddress[0].country : ''}
+              onChange={handleAddressChange}
+            >
+              <option value="India">India</option>
+            </Form.Control>
+          </Form.Group>
+        </Form.Row>
+        <Form.Row>
+          <Form.Group as={Col} xs={8} md={6}>
+            <Button 
+              variant="outline-primary" 
+              size="lg" 
+              block
+              onClick={updateHospitalAddress}
+            >
+              Update Info
+            </Button>{' '}
+          </Form.Group>
+        </Form.Row>
+      </Form>
+
+      <Form>
+        <Form.Row>
+          <Form.Label>Filter By:</Form.Label>
+          <Form.Group as={Col} controlId="formGridState">
+            <Form.Control as="select" value="Choose...">
+              <option>State</option>
+              <option>...</option>
+            </Form.Control>
+          </Form.Group>
+
+          <Form.Group as={Col} controlId="formGridState">
+            <Form.Control as="select" value="Choose...">
+              <option>District</option>
+              <option>...</option>
+            </Form.Control>
+          </Form.Group>
+
+          <Form.Group as={Col} controlId="formGridState">
+            <Form.Control as="select" value="Choose...">
+              <option>India</option>
+            </Form.Control>
+          </Form.Group>
+        </Form.Row>
+      </Form>
+
+      <Form>
+        <h5>Fortis Hospital</h5>
+        <Form.Text className="text-muted">
+          1 June 2020, 05:14:33am
+        </Form.Text>
+        <Form.Row>
+          {inventoryList.map((item, index) => (
+            <Card 
+              className={`text-center ${style.inputCard}`}
+              key={item.resourceType} >
+                <Form.Text>
+                  <strong>{item.resourceType}</strong>
+                </Form.Text>
+                <Form.Group>
+                  <Form.Control
+                    type="number"
+                    className={style.inventoryInput}
+                    data-index={index}
+                    value={item.available ? item.available : 0}
+                    onChange={handleInventoryChange}
+                  />
+                </Form.Group>
+            </Card>
+          ))}
+        </Form.Row>
+        <Form.Row>
+          <Form.Label column md={2}></Form.Label>
+          <Form.Group as={Col} md={8}>
+            <Button 
+              variant="outline-primary" 
+              size="lg" 
+              block
+              onClick={updateInventoryList}
+            >
+              Update Inventory
+            </Button>{' '}
```

Comment:
I don't think that you need to force a space between closing a button and closing a form group.